OktaIdxExample/OktaIdxExampleUITests/Utilities/Default Actions/CreateUser.swift

```diff
@@ -14,7 +14,7 @@ import Foundation
 import OktaSdk
 
 extension ScenarioValidator {
-    func createUser(username: String, password: String, firstName: String, lastName: String, completion: @escaping (Error?) -> Void) {
+    func createUser(username: String, password: String, firstName: String, lastName: String, groupNames: [String] = [], completion: @escaping (Error?) -> Void) {
```

Comment:
For simplicity, I'd probably call this `groups`, and pass an array of enums.  Then, we can define our own group names as an enums so the actual names are centralized.

Reply:
Great idea!